src/Apps/Home/Components/HomeContentCards/HomeContentCards.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useRef, useState } from "react"
+import { CaptionedImage as BrazeContentCard } from "@braze/web-sdk"
+import { BrazeCards } from "./BrazeCards"
+import { FallbackCards } from "./FallbackCards"
+import { PlaceholderCards } from "./PlaceholderCards"
+import { useRouter } from "System/Router/useRouter"
+import { paramsToCamelCase } from "Components/ArtworkFilter/Utils/urlBuilder"
+
+const sortCards = (lhs, rhs) => {
+  const lhsPosition = (lhs.extras || {}).position || lhs.id
+  const rhsPosition = (rhs.extras || {}).position || rhs.id
+  return lhsPosition > rhsPosition ? 1 : -1
+}
+
+const DEFAULT_TIMEOUT_AMOUNT = 1000
```

Comment:
`non-blocking`: I *suspect* this default will wind up being a bit too aggressive. I'd probably opt for something closer to like 5 seconds.

Reply:
I don't have strong feelings here so I'm happy to move to 5 seconds. 👍